ISSUE_TEMPLATE.md

```diff
@@ -1,6 +1,6 @@
-Hello! Please read the contributing guidelines before submitting an issue.
+Hello! Please read the _contributing guidelines_ before submitting an issue regarding the __GitHub Extension for Visusal Studio__.
```

Comment:
There's not much point in doing italics or bold in the templates, they're not going to be rendered when people are filling up the information.